homeassistant/components/zwave_js/event.py

```diff
@@ -0,0 +1,109 @@
+"""Support for Z-Wave controls using the event platform."""
+from __future__ import annotations
+
+from zwave_js_server.client import Client as ZwaveClient
+from zwave_js_server.model.driver import Driver
+from zwave_js_server.model.value import Value, ValueNotification
+
+from homeassistant.components.event import DOMAIN as EVENT_DOMAIN, EventEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import ATTR_VALUE, DATA_CLIENT, DOMAIN
+from .discovery import ZwaveDiscoveryInfo
+from .entity import ZWaveBaseEntity
+
+PARALLEL_UPDATES = 0
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Z-Wave Event entity from Config Entry."""
+    client: ZwaveClient = hass.data[DOMAIN][config_entry.entry_id][DATA_CLIENT]
+
+    @callback
+    def async_add_event(info: ZwaveDiscoveryInfo) -> None:
+        """Add Z-Wave event entity."""
+        driver = client.driver
+        assert driver is not None  # Driver is ready before platforms are loaded.
+        entities: list[ZWaveBaseEntity] = []
+        if info.platform_hint == "stateless":
```

Comment:
Do we need this check? There's no other case now.

Reply:
not necessarily, it was to act as a fallback but will remove 👍🏾